internal/io/file/constant.go

```diff
@@ -22,6 +22,13 @@ const (
 	LINES_TYPE FileType = "lines"
 )
 
+const (
+	ZLIB          = "zlib"
+	GZIP          = "gzip"
+	FLATE         = "flate"
+	NONE_COMPRESS = "none"
```

Comment:
What does `NONE_COMPRESS` stand for?

Reply:
- I think all these zlib/flate/gzip are the same compression algorithm. We can leave only gzip, and maybe add zstd
- Any reason not to use empty string "" as none. It is the default string init value.